i18n/strings.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/* THIS IS A GENERATED FILE. DO NOT EDIT DIRECTLY */
+$i18nStrings = array(
+"Saving…" => __( "Saving…" ),
```

Comment:
I used the same code as Calypso, so the better solution would be to modify ```preprocess-xgettextjs-match.js``` inside ```wp-calypso``` to use single quotes.

Also, a tab is easy to add, I'll do it, but let's not forget that this is an auto-generated file and shouldn't be edited or even read by humans.